oap-server/server-query-plugin/debugging-query-plugin/src/main/java/org/apache/skywalking/oap/query/debug/DebuggingHTTPHandler.java

```diff
@@ -160,8 +160,8 @@ public String execExpression(@Param("dumpDBRsp") boolean dumpStorageRsp,
 
     @SneakyThrows
     @Get("/debugging/query/trace/queryBasicTraces")
-    public String queryBasicTraces(@Param("service") String service,
-                                   @Param("serviceLayer") String serviceLayer,
+    public String queryBasicTraces(@Param("service") Optional<String> service,
```

Comment:
Based on the implementation I think `service` and `serviceLayer` are required and should not be `Optional`

Reply:
The `serviceId/serviceInstanceId/endpointId` in the graphQL are optional, the implement also can ignore them.
https://github.com/apache/skywalking-query-protocol/blob/6ed9a1bb96c1c753d3211e55995117c9617e4339/trace.graphqls#L37